libcloud/test/compute/fixtures/gce/global_backendServices_web_service.json

```diff
@@ -7,14 +7,14 @@
  "backends": [
   {
    "description": "",
-   "group": "https://www.googleapis.com/resourceviews/v1beta1/projects/project_name/zones/us-central1-b/resourceViews/us-resources",
+   "group": "https://content.googleapis.com/compute/v1/projects/project_name/zones/us-central1-a/instanceGroups/myinstancegroup",
```

Comment:
Really? The URL has *content* vs *www*?  That seems like something we should look investigate internally.